ci/do_ci.sh

```diff
@@ -216,6 +216,7 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
     --define quiche=enabled \
     --define path_normalization_by_default=true \
     --define deprecated_features=disabled \
+    --define use_legacy_codecs_in_test=true \
```

Comment:
should we call this out in bazel/README.md ?  It's our first test-only options so I'm good either way.

Reply:
If we're aiming for 4 weeks I think I just don't care - feel free to mark this one resolved :-)